cli/pkg/providers/aws/aws.go

```diff
@@ -49,42 +49,45 @@ func (p *provider) GetOverrides() tmpl.Variables {
 	}
 }
 
-func (p *provider) PostInstallation(dir workspace.ConfigDir) error {
-	conf, err := dir.LoadAiryYaml()
-	if err != nil {
-		return err
-	}
+func (p *provider) PostInstallation(providerConfig map[string]string, dir workspace.ConfigDir) error {
+	if providerConfig["hostUpdate"] != "false" {
```

Comment:
We control with a flag whether to update the hosts after the installation or `global.kubernetes.host` is used in the ingress definitions and in the `hostnames` configMap.